templates/includes/sidebar.html

```diff
@@ -1,4 +1,12 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: {{ user.username }}</li>
```

Comment:
you skipped this step

![image](https://user-images.githubusercontent.com/50211819/226679148-b98393c0-a82b-43c7-84e5-2d218b01cd0c.png)